modules/widgets/simple-payments.php

```diff
@@ -282,6 +297,11 @@ public function validate_ajax_params( $params ) {
 				$errors->add( 'price', __( 'Everything comes with a price tag these days. Please add a your product price.', 'jetpack' ) );
 			}
 
+			$precision = strcmp( $params['currency'], 'JPY' ) === 0 ? 0 : 2;
```

Comment:
if we are going to assert that Japanese Yen is the only currency that doesn't use two decimal places then I think we need a comment explaining that.

http://apps.cybersource.com/library/documentation/sbc/quickref/currencies.pdf